dc/core/MasterToWallChannel.h

```diff
@@ -76,19 +76,19 @@ public slots:
      * Send the given DisplayGroup to the wall processes.
      * @param displayGroup The DisplayGroup to send
      */
-    void sendAsync( DisplayGroupPtr displayGroup );
+    void sendDisplayGroup( DisplayGroupPtr displayGroup );
```

Comment:
Are these name changes absolutely necessary for the change of connect() notation? Because send() and sendAsync() were used to distinguish very different behaviours, see documentation at line 59...